hermes/bin/src/runtime_extensions/hermes/logging/log_msg.rs

```diff
@@ -0,0 +1,53 @@
+//! Implementation of logging API
+use tracing::info;
+
+/// Log a message
+pub(crate) fn log_message(
+    ctx: Option<String>, msg: &str, file: Option<String>, function: Option<String>,
+    line: Option<u32>, col: Option<u32>, data: Option<String>,
+) {
+    // Force the log level to be info
+    info!(
+        ctx = ctx.unwrap_or_default(),
+        message = msg,
+        file = file.unwrap_or_default(),
```

Comment:
there needs to be a field for the level the wasm assigned to the log, so we can use it in the subscribers to properly classify them.